docs/diagrams/model/ModelClassDiagram.puml

```diff
@@ -30,25 +37,35 @@ AddressBook .up.|> ReadOnlyAddressBook
 
 ModelManager .up.|> Model
 Model .right.> ReadOnlyUserPrefs
-Model .left.> ReadOnlyAddressBook
+Model .down.> ReadOnlyAddressBook
 ModelManager -left-> "1" AddressBook
 ModelManager -right-> "1" UserPrefs
 UserPrefs .up.|> ReadOnlyUserPrefs
 
 AddressBook *--> "1" UniquePersonList
+AddressBook *--> "1" UniqueStudentList
 UniquePersonList --> "~* all" Person
 Person *--> Name
 Person *--> Phone
 Person *--> Email
 Person *--> Address
 Person *--> "*" Tag
+Student --|> Person
+UniqueStudentList --> "~* all" Student
+Student *--> TutorialGroup
+Student *--> StudentNumber
+Student *--> Assignment
```

Comment:
Student should also include the List of AttendanceRecord, with the new Attendance Model. Other than that, LGTM.